katcp/kattypes.py

```diff
@@ -707,8 +707,6 @@ def raw_handler(self, msg):
 
     return decorator
 
-# Using partial with no extra parameters lets us 'copy' the function so that we
-# can change the docstring without affecting the original function's docstring
```

Comment:
Ok, but I guess modifying it to remove the "with no extra parameters" part would make it true. The comment-worthiness for me is the 'docstring override' part.

But anyway, no biggy.